README.md

```diff
@@ -50,6 +50,29 @@ server.post(
 server.listen(0)
 ```
 
+Alternatively, `@fastify/session` is also supported and works out of the box for session storage.  
```

Comment:
Adding a link would be great. What's the difference between @fastify/session and fastify-secure-session? Can you document it?

Reply:
I have added the link and some differences. Please review to see if it makes sense.